website/src/config-reference.md

```diff
@@ -397,6 +397,23 @@ public function doFoo(): ?string
 }
 ```
 
+### `checkTooWideParameterOutInProtectedAndPublicMethods`
+
+**default**: `false`
```

Comment:
Should have the green "Available in ..." that can be seen here https://phpstan.org/config-reference#pollutescopewithblock